lib/tasks/stress_create_msgs_signatures_file.rake

```diff
@@ -0,0 +1,32 @@
+require 'task_helpers/stress_sms_helper'
+
+namespace :stress do
+  desc "Generate a csv file with msg,number and signature for JMeter test"
+  task :create_msgs_signatures_file, [:auth_token,
```

Comment:
I want to be sure the knowledge of how to run these tests is captured. Can you write a quick .md file in the docs/ directory on how to do so?